client/src/components/Interview/Interview.js

```diff
@@ -1,83 +1,86 @@
 import React from "react"
 import "./Interview.css"
+import QuestionAnswer from "./subcomponents/QuestionAnswer"
```

Comment:
Maybe it's just me, but here it's `QuestionAnswer` but the actual js file name is `questionAnswer` so it directs to a wrong file.